### PR TITLE
Add empty pensions app

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2879

Adds the empty pensions app, based on hca.

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -125,6 +125,7 @@ const ignore = require('metalsmith-ignore');
 const ignoreList = [];
 if (options.buildtype === 'production') {
   ignoreList.push('healthcare/rjsf/*');
+  ignoreList.push('pensions/application.md');
```
Comment (cvalarida) [inline script/build.js]: What does this ignore list do, exactly?

File: src/js/pensions/pensions-entry.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+
+import initReact from '../common/init-react';
+import route from './routes';
+import createCommonStore from '../common/store';
+import createLoginWidget from '../login/login-entry';
+import reducer from './reducer';
+
+require('../common');
```
Comment (cvalarida) [inline src/js/pensions/pensions-entry.jsx]: Ohh, interesting. `require()`ing to inject all the js from `../common/index.js`, aye? I was a bit befuddled why we were mixing `import` and `require()`, but I think I get it.

I imagine we'd only ever do this at the entry point, yes?

File: src/sass/pensions.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Variables
+
+@import "style";
+@import "modules/m-process-list";
```
Comment (cvalarida) [inline src/sass/pensions.scss]: Isn't this the subway map? Will we need this module?

**Review by cvalarida — APPROVED** [review]
Had some questions, but it looks good as a scaffold.

File: script/build.js (hunk 1 above)
Comment (jbalboni) [inline script/build.js]: It keeps us from generating an html page at `pensions/application` in production.

File: src/js/pensions/pensions-entry.jsx (hunk 2 above)
Comment (jbalboni) [inline src/js/pensions/pensions-entry.jsx]: We should be able to use import here, I just copied it from other entry pages.

File: src/sass/pensions.scss (hunk 3 above)
Comment (jbalboni) [inline src/sass/pensions.scss]: I am assuming so for the intro page.

Comment (cvalarida) [inline src/sass/pensions.scss]: Ahh, the intro page. Okay, cool.

File: src/js/pensions/pensions-entry.jsx (hunk 2 above)
Comment (cvalarida) [inline src/js/pensions/pensions-entry.jsx]: Ah, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) says we can `import '../common` "for side-effects only". Okay.

:man_shrugging:

File: script/build.js (hunk 1 above)
Comment (cvalarida) [inline script/build.js]: Ahhh, that's why I don't see the edu forms in there. Awesome! Thanks for the clarification. 😄

**Pull request merged**